### PR TITLE
update for checking friend's playlists

### PR DESCRIPTION
**Pull request by Starliyiyu**
for checking friend's playlists:
implemented use cases, controller, view (to be integrated)
moved enum Privacy to a seperate class

Dsgateway to Playlist DB TBA

### CODE REVIEW

**Review by varygx — APPROVED** [review]
LGTM

File: src/main/java/entities/playlist_entities/Playlist.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import entities.Song;
 import java.util.ArrayList;
 
+;
```
Comment (cynthiaxsa) [inline src/main/java/entities/playlist_entities/Playlist.java:7]: floating semicolon

**Review by cynthiaxsa — APPROVED** [review]
Looks solid! Go forth and continue coding progress, the comments you have on here are very informative as to what your code is doing

**Review by wilso16 — APPROVED** [review]
Make sure cluttered/long code is avoided. Also javadocs 😅

**Pull request merged**